mobile/lib/modules/backup/background_service/background.service.dart

```diff
@@ -429,9 +437,22 @@ class BackgroundService {
     return "$percent% ($_uploadedAssetsCount/$_assetsToUploadCount)";
   }
 
-  void _onAssetUploaded(String deviceAssetId, String deviceId) {
+  void _onAssetUploaded(
+    BackupService backupService,
+    String deviceAssetId,
+    String deviceId,
+    bool isDuplicated,
+  ) {
     debugPrint("Uploaded $deviceAssetId from $deviceId");
-    _uploadedAssetsCount++;
+
+    if (isDuplicated) {
+      backupService.saveDuplicatedAssetIdToLocalStorage(deviceAssetId);
+      _assetsToUploadCount--;
```

Comment:
@zoodyy I tried to write this to Hive in background backup but it doesn't seem to work, any suggestion from your side?

Reply:
`_onAssetUploaded ` is only called when the user enabled showing progress... so it will currently not save duplicated IDs if those notifications are disabled. Will provide a fix in moment.
Edit: fixed